specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/gallery.json

```diff
@@ -2966,6 +3607,248 @@
       ],
       "description": "The definition of a parameter that can be passed to a custom action of a Gallery Application Version."
     },
+    "GalleryScript": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/GalleryScriptProperties"
```

Comment:
>Resource [](http://example.com/codeflow?start=69&length=8)

Is it possible to choose either of TrackedResource or ProxyResource ?

Reply:
I had changed Resource to TrackedResource. I had to revert it due to below error:

<img width="878" alt="{C3A466B5-9666-47CB-86A3-339BB391F9FB}" src="https://github.com/user-attachments/assets/fa53a65b-9d30-4e8b-87b0-d8e517d59244" />